sys/linux/socket_netlink_route_sched.txt

```diff
@@ -76,11 +90,37 @@ type tca_kind_options_t[NAME, VALUES] {
 	TCA_OPTIONS	nlattr[TCA_OPTIONS, VALUES]
 }
 
+type tca_kind_comp_t[NAME, OPTION, VALUES] {
+	TCA_KIND	nlattr[TCA_KIND, string[NAME]]
+	TCA_OPTIONS	nlattr[TCA_OPTIONS, OPTION]
+	TCA_OPTIONS	nlattr[TCA_OPTIONS, VALUES]
```

Comment:
Don't we error on duplicate field names? We should. Please name them differently.

Reply:
We indeed allowed duplicate field names in template structs. This is fixed now:
https://github.com/google/syzkaller/commit/256b70f9cfff65292358cf8c0b9d2611db56c326
https://github.com/google/syzkaller/commit/90c54c496b268f0977517448e8f7de54e93b975d